addon/adapters/drf.js

```diff
@@ -122,12 +122,13 @@ export default DS.RESTAdapter.extend({
         payload[key].forEach((error) => {
           if (key === this.get('nonFieldErrorsKey')) {
             out.push({
-              meta: {key: key},
-              detail: error
+              source: { pointer: 'data' },
+              detail: error,
+              title: 'Validation Error'
             });
           } else {
             out.push({
-              source: { pointer: `data/attributes/${key}`},
+              source: { pointer: `data/attributes/${key}` },
```

Comment:
Ember Data 1.13.7 (released yesterday) "fixed" the pointer format to align with the JSON API spec.

https://github.com/emberjs/data/pull/3558

I think you'll need to add a forward slash to the beginning here (and on line 125), and update the documentation to show compatibility with ED >= 1.13.7